### PR TITLE
Redhat support

### PR DESCRIPTION
**Pull request by solreynolds**
Adds redhat support.

### CODE REVIEW

**Comment by solreynolds** [comment]
The vagrant subfolder contains some submodules - This branch uses a forked Mayeu.RabbitMQ submodule and will need to be updated with the fixed version.

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,9 @@
-[submodule "vagrant/roles/Mayeu.RabbitMQ"]
-	path = vagrant/roles/Mayeu.RabbitMQ
-	url = https://github.com/Mayeu/ansible-playbook-rabbitmq
 [submodule "vagrant/roles/redis"]
 	path = vagrant/roles/redis
-	url = https://github.com/Mayeu/ansible-playbook-redis
+	url = https://github.com/geerlingguy/ansible-role-redis.git
```
Comment (SydOps) [inline .gitmodules]: I don't think the owner of this repo will be happy for this change.

File: .gitmodules [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+[submodule "vagrant/roles/Mayeu.RabbitMQ"]
+	path = vagrant/roles/Mayeu.RabbitMQ
+	url = https://github.com/solreynolds/ansible-playbook-rabbitmq.git
```
Comment (SydOps) [inline .gitmodules]: I don't think the owner of this repo will be happy for this change.

File: tasks/common_redhat.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+- copy: src=sensu.repo dest=/etc/yum.repos.d/
+        owner=root group=root mode=0644
+
+- yum: name={{ item }} state=present
+  with_items:
+  - ruby
+  - ruby-devel
+  - rubygems
+  - rubygem-aws-sdk
```
Comment (SydOps) [inline tasks/common_redhat.yml]: Is this must-have package?

**Comment by Mayeu** [comment]
Thank you for your contribution.

Sadly, and has you may have guessed, I don't use this project anymore, nor did I have the time to continue to maintain it.

You can now find an official playbook maintained at [sensu/sensu-ansible](https://github.com/sensu/sensu-ansible).

Wishing you the best, and a gajillion more contribution in your lifetime  :pray::sparkles:

**Pull request closed without merging**